micromamba/tests/test_create.py

```diff
@@ -613,6 +613,17 @@ def test_spec_with_channel(tmp_home, tmp_root_prefix, tmp_path):
             assert link["version"].startswith("0.22.")
 
 
+def test_spec_with_channel_and_subdir():
```

Comment:
We should extend this test to make sure `pkgs/main::package_name` works correctly.